tests/sriov/README.md

```diff
@@ -0,0 +1,221 @@
+# SRIOV Basic Tests
```

Comment:
Could we write it so that if I read the top part of the readme, I immediately know what I want to know instead of having to read two whole doc?

Could we start by stating how to run the tests?

Title is "basic tests". What are basic tests, what are non-basic tests?